### PR TITLE
add clr file support for IDA >= 7.0

### PR DESCRIPTION
**Pull request by Jinmo**
This pull request adds loading of CLR files specified by manifest.json.

### CODE REVIEW

**Comment by athre0z** [comment]
Wow, very cool, thank you very much for this! I'll review this immediately.

File: plugins/idaskins/clrapplier.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    def __init__(self, **kwargs):
+        self.hooks = {NativeHook.NAMES[key]: value for key, value in kwargs.items()}
+        self._handler = hook_cb_t(self.handler)
+
+    def handler(self, _user_data, code, _va_args):
+        if code in self.hooks:
+            try:
+                res = self.hooks[code]()
+                return cast(res, c_void_p).value
+            except:
```
Comment (athre0z) [inline plugins/idaskins/clrapplier.py]: Can this except statement be a bit more specific, e.g. just catch `KeyError`?

Comment (Jinmo) [reply plugins/idaskins/clrapplier.py]: In this line, I tried to return NULL pointer if any exception has occurred. I just tried `raise Exception()` outside try-except and IDA stuck in an infinite loop which is weird behavior...

**Comment by Jinmo** [comment]
Added a fix for mac not having `WINFUNCTYPE`, exception handler when CLR file is missing, and traceback printer.

File: plugins/idaskins/clrapplier.py (hunk 1 above)
Comment (athre0z) [reply plugins/idaskins/clrapplier.py]: Alright, I see! IDA's Py embedding is not always perfect, printing a stack-trace is probably the best we can do here.

**Comment by athre0z** [comment]
Excellent, merged!

**Pull request merged**